src/compiler/commandLineParser.ts

```diff
@@ -716,7 +716,8 @@ namespace ts {
                         }
 
                         // Skip over any minified JavaScript files (ending in ".min.js")
-                        if (/\.min\.js$/.test(fileName)) {
+                        // Skip over dotted files and folders as well
+                        if (/\.min\.js$/.test(fileName) || /[\\/]\.[\w.]/.test(fileName)) {

```

Comment:
also can you add a unit test. something along the lines of https://github.com/Microsoft/TypeScript/blob/master/tests/cases/unittests/convertCompilerOptionsFromJson.ts